### DISCUSSION TITLE
Hello Woocommerce I want to cancel or override the postcode section

### DISCUSSION THREAD

**Discussion by ozgureis** [woocommerce/woocommerce#52190] [Checkout Flow]
I want to cancel or override the postcode section

![image](https://github.com/user-attachments/assets/1a344ad8-e70c-41de-b76e-7c1a07fe7e12)

**Comment by nielslange** [comment]
Hello @ozgureis 👋

What do you mean by overwriting the postcode selection? By default, when a new customer is purchasing something from your store, they do not see any address data. Address data only appears for returning customers, or for you, when being logged in. When changing the postcode in the checkout form and placing a new order, the postcode, that's stored for this account, will also be updated in the backend. 

Does that answer your question, or were you looking for something else?
